config/base/catalogd/rbac/role.yaml

```diff
@@ -4,6 +4,14 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Can this set of permissions be moved to a Role? Looks like that's how we have it setup for operator-controller. The image pull secrets attached to the service account are in our pod's namespace.

Reply:
Hmm.... I'm not 100% sure it can, because it needs to be able to access the global pull secret which could be in any namespace. But I suppose that's more of a downstream issue, as the default deployment upstream does not supply a global pull secret, and users can always add the proper permissions.